server.js

```diff
@@ -167,14 +179,28 @@ function renderPage(appHtml, reactHelmet, canonicalUrl) {
           ``
         )}rss/latest">
         ${reactHelmet.title.toString()}
-        <meta name="ga-identifier" content="UA-87658599-4">
-        <meta name="gtm-identifier" content="GTM-KHLX47C">
+        <meta name="ga-identifier" content="${GA_ID}">
+        <meta name="gtm-identifier" content="${GTM_ID}">
         ${recaptcha}
       </head>
       <body>
         <div id="app">${appHtml}</div>
         <script type="application/json" id="environment-variables">${envUtilities.serializeSafeEnvAsJSON()}</script>
         <script src="/bundle.js"></script>
+        <script nonce="${cspNonce}">
+        (function (w, d, s, l, i) {
+          w[l] = w[l] || [];
+          w[l].push({ "gtm.start": new Date().getTime(), event: "gtm.js" });
+          var f = d.getElementsByTagName(s)[0],
+            j = d.createElement(s),
+            dl = l != "dataLayer" ? "&l=" + l : "";
+          j.async = true;
+          j.src = "https://www.googletagmanager.com/gtm.js?id=" + i + dl;
+          var n = d.querySelector("[nonce]");
+          n && j.setAttribute("nonce", n.nonce || n.getAttribute("nonce"));
+          f.parentNode.insertBefore(j, f);
+        })(window, document, "script", "dataLayer", "${GTM_ID}");
+        </script>
```

Comment:
This basically is the same script snippet included in the GDoc but in an easier to read format.